go.mod

```diff
@@ -8,6 +8,8 @@ exclude k8s.io/kubernetes v1.13.0
 // Client-go does not handle different versions of mergo due to some breaking changes - use the matching version
 replace github.com/imdario/mergo => github.com/imdario/mergo v0.3.5
 
+replace istio.io/api => github.com/briansonnenberg/api v0.0.0-20221004175107-63c6b7b3c0ae
```

Comment:
"github.com/briansonnenberg/api" seems to be a typo.

Reply:
Yes, this was just used for my testing.  Will address.